Testing/functional/e2e/cypress/fixtures/ImmunizationService/immunization.json

```diff
@@ -478,7 +478,7 @@
     },
     {
       "id": "890700884_54396874",
-      "name": "Pringles",
+      "name": "COVID-19",
```

Comment:
Is this needed to fix the bug with comments as it will conflict with my Immunization functional tests.

Reply:
should not conflict as it is the same data